### PR TITLE
adig: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

- Deprecate `specified` in favor of `specific`
- Warning: can't understand `A` semantic: is it an wrong-cased article or some name?

### CODE REVIEW

File: pages/linux/adig.md [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
 
 `adig -s {{1.2.3.4}} {{example.com}}`
 
-- Use specified TCP port to connect to DNS server:
+- Use a specific TCP port to connect to DNS server:
 
 `adig -T {{port}} {{example.com}}`
 
-- Use specified UDP port to connect to DNS server:
+- Use a specific UDP port to connect to DNS server:
```
Comment (kbdharun) [inline pages/linux/adig.md:22]:
```suggestion
- Use a specific UDP port to connect to the DNS server:
```

File: pages/linux/adig.md [hunk 2]
```diff
@@ -11,14 +11,14 @@
 
 `adig -d {{example.com}}`
 
-- Connect to [s]pecified DNS server:
+- Connect to a specific DNS server:
 
 `adig -s {{1.2.3.4}} {{example.com}}`
 
-- Use specified TCP port to connect to DNS server:
+- Use a specific TCP port to connect to DNS server:
```
Comment (kbdharun) [inline pages/linux/adig.md:18]:
```suggestion
- Use a specific TCP port to connect to the DNS server:
```

Comment (EmilySeville7cfg) [reply pages/linux/adig.md:18]: I guess `a` is better here as more than one DNS server can be connected to.

Comment (kbdharun) [reply pages/linux/adig.md:18]: Yeah, that works too. Updated the suggestions.

File: pages/linux/adig.md [hunk 3]
```diff
@@ -11,14 +11,14 @@
 
 `adig -d {{example.com}}`
 
-- Connect to [s]pecified DNS server:
+- Connect to a specific DNS server:
```
Comment (blueskyson) [inline pages/linux/adig.md:14]:
```suggestion
- Connect to a specific DNS [s]erver:
```

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**